localstack/services/ses/provider.py

```diff
@@ -128,7 +128,10 @@ class SesServiceApiResource:
     - GET param `email`: filter for `source` field in SES message
     """
 
-    def on_get(self, request):
+    def on_get(self, request, message_id=None):
```

Comment:
I would make `message_id` a query parameter which is more backward compatible.

Could we also update the docstring? Also the behaviour when both `message_id` and `email` are provided.